theano/gpuarray/tests/check_dnn_doc.py

```diff
@@ -0,0 +1,284 @@
+"""
```

Comment:
Why is this file called check_dnn_doc if it is a collection of helper functions?